src/managers/ProjectManager.ts

```diff
@@ -766,4 +811,25 @@ export class ComponentLibraryProject extends Project {
             }
         });
     }
+
+    public async publish() {
+        if(this.libType){
+            if(this.libType !== 'channelstore') {
+                util.log(`No publish for libType: ${this.libType}`)
```

Comment:
Many devs will be using the `'remote'` style of channel store and don't want to see this log all the time, so let's utilize `this.logger.debug` instead of `util.log`